pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -299,13 +298,28 @@ func (lbaas *LbaasV2) GetLoadBalancer(clusterName string, service *api.Service)
 	return status, true, err
 }
 
+func nodeAddressForLB(node *api.Node) (string, error) {
```

Comment:
This is in-between both. If you want to make some policy clear, actually mention the policy in a comment. Saying the address is ForLB doesn't imply that the ip must be on the same subnet.